### PR TITLE
Bugfix: explicitly attach gas, fix indexer url

### PR DESCRIPTION
**Pull request by judith-Near**

### CODE REVIEW

File: src/store/multisafe/thunks/onEditMultisafe.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { thunk } from 'easy-peasy';
+import BN from 'bn.js';
 import { signTransactionByLedger } from '../helpers/signTransactionByLedger';
 import { getRoute } from '../../../ui/config/routes';
 
-// TEN PLIK PRZENIESC do multisafe/thunks
```
Comment (marcinbodnar) [inline src/store/multisafe/thunks/onEditMultisafe.js:5]: Thanks :)

**Review by marcinbodnar — APPROVED** [review]

**Comment by mattlockyer** [comment]
@judith-Near @marcinbodnar can we create some sort of UI parameter for this?

And 40 Tgas won't cover a lot of the outlying edge cases to date. We'd need the default to be something more like 100 Tgas.

Perhaps there's also a way you can RPC view call the request and extract the gas specified if the request is a functionCall and then add an additional 10 Tgas on top to handle the promise call and return gas requirements?

Happy to discuss more, but I think a use customizable gas amount would be the first step, so they can confirm with a higher user preference gas amount before encountering errors on the NEAR Wallet.

**Comment by marcinbodnar** [comment]
@mattlockyer  Agree. We can treat this PR as a hot fix, to unblock the partner and discuss additional solutions separately. Is that ok with you?

**Comment by agileurbanite** [comment]
@marcinbodnar i don't think hard coding this in here makes sense. @judith-Near can we work with @Patrick1904 as I believe we can estimate the gas requirement.

**Comment by mattlockyer** [comment]
> @mattlockyer Agree. We can treat this PR as a hot fix, to unblock the partner and discuss additional solutions separately. Is that ok with you?

The partner needs at least 60 Tgas. TBH I think 100 Tgas is not even that bad and covers a lot of use cases. e.g. everything in apps like ref, burrow etc... require 100 Tgas.

**Review by judith-Near — COMMENTED** [review]
@mattlockyer im curious, why would someone need more gas to call "confirm" in the multisig2 contract? ie "confirm" typically uses 8 TGas, "add_request_and_confirm" uses 15 TGas. why would someone need 100TGas for these calls?

**Comment by mattlockyer** [comment]
> @mattlockyer im curious, why would someone need more gas to call "confirm" in the multisig2 contract? ie "confirm" typically uses 8 TGas, "add_request_and_confirm" uses 15 TGas. why would someone need 100TGas for these calls?

Because the functionCall they are confirming requires at least 50Tgas.

It's not simply the gas cost of `confirm` that you must take into consideration. The request, if confirmed, creates a new promise, and if it is a functionCall specifying 50 Tgas, it will attempt to attach that much Tgas.

See the request payload here: https://explorer.testnet.near.org/transactions/HgPJLQNfE1o3FEGwJGGCPF2KfxnPcrFDs2DWwJTyAJLr

If there's no UI option to specify the Tgas attached to confirm and it defaults to 30 or 40 Tgas, requests like the above will always fail and the user will have to retry using the wallet logic, this is simply bad UX.

Given there are a LOT of situations where someone might want to use multisafe TXs to confirm functionCall requests that require 50+ Tgas, I think you should temporarily increase to at least 100Tgas and longer term you can either:

1. Check the request they are confirming and if the gas field is present you can ADD this amount to the default gas amount.
2. Provide a UI option to specify the gas.

But you cannot continue to send users to the wallet with a fixed default gas amount and no option to increase, because you are sending them into a preventable error and retry scenario. This is bad UX.

**Review by marcinbodnar — APPROVED** [review]

**Pull request merged**